experimental/displaynames/src/displaynames.rs

```diff
@@ -68,12 +75,12 @@ impl DisplayNames {
         ]
     );
 
-    /// Returns the display name of the region for a given string.
+    /// Returns the locale specific display name of a region for a given string.
     /// This function is locale-sensitive.
-    pub fn of(&self, region_code: &str) -> Option<&str> {
-        match <TinyAsciiStr<3>>::from_str(region_code) {
+    pub fn of(&self, code: &str) -> Option<&str> {
+        match <TinyAsciiStr<3>>::from_str(code) {
             Ok(key) => {
-                let data = self.data.get();
+                let data = self.region_data.get();
                 match self.options.style {
                     Style::Short => data.short_names.get(&key),
                     _ => data.names.get(&key),
```

Comment:
Note: Both here and below, we should do the fallback from short to long automatically. We could optionally have another function, like `of_strict`, that does not fall back

Reply:
Added the change to fallback to the names without any alt variants. I can add a function of_strict later if needed.